### ISSUE TITLE
Add support for data and map files

### ISSUE THREAD

**Issue by Abbe98** [yarl/pattypan#66]
Is uploading content to the Data namespace out of scope?

**Comment by yarl** [comment]
Sounds good. As long it can be uploaded via `action=upload` we can support it. Do you have an example of uploaded data like this?

**Comment by Abbe98** [comment]
The data namespace works quite differently from the File namespace. Every file is actually a page with it's content as GeoJSON, JSON or CSV. So uploading a Data file would include(simplified):

 - Validating the file, and check for required fields.
 - Creating the page.

It's a quite large task but there exist no import solutions except for scripts and I think GLAMs would benefit from not having a separate tool...

**Comment by yarl** [comment]
I agree. If you have links do specs it will be good to store them here.

**Comment by Abbe98** [comment]
My bad:

Tabular Data (JSON/CSV):
https://www.mediawiki.org/wiki/Help:Tabular_Data

Map Data (GeoJSON):
https://www.mediawiki.org/wiki/Help:Map_Data
